src/bpf_writer.cc

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include <elfio/elfio.hpp>
+
+#include "bpf_writer.h"
+
+const std::string symbol_section_name = ".symtab";
+const std::string maps_section_name = "maps";
```

Comment:
Be aware that the "maps" section is considered obsolete on Linux, whereas the BTF ".maps" section is current.
https://github.com/dthaler/ebpf-docs-1/blob/elf/rst/elf.rst#data-sections talks about this, but is still missing the details on the ".maps" section.

Reply:
Added note and issue to track this.